l10n_it_fatturapa_out/wizard/wizard_export_fatturapa.py

```diff
@@ -57,6 +58,26 @@ class WizardExportFatturapa(models.TransientModel):
     _name = "wizard.export.fatturapa"
     _description = "Export FatturaPA"
 
+    @api.model
+    def _domain_ir_values(self):
+        """Get all print actions for current model"""
+        return [('model', '=', self.env.context.get('active_model', False)),
+                ('key2', '=', 'client_print_multi')]
+
+    @api.model
+    def _default_ir_values(self):
```

Comment:
@SimoRubi di default lo lascerei vuoto. Normalmente l'utente esporta l'XML con i soli dati

Reply:
@eLBati eliminato il default in https://github.com/OCA/l10n-italy/pull/619/commits/a294da6b2f488fc9b6a9c81773a16431787f82fd